ohbmbrainhack2023/vox2cortex/1.0.0/spec.yml

```diff
@@ -0,0 +1,41 @@
+#### container info
+image:
+  singularity: nobrainer-zoo_vox2cortex.sif
+  docker: neuronets/nobrainer-zoo:vox2cortex
+
+#### repository info
+repository:  
+    repo_url: "None"
+    committish: "None"
+    repo_download: False
+    repo_download_location: "None"
+    
+#### required fields for prediction
+inference:
+    prediction_script: "trained-models/ohbmbrainhack2023/vox2cortex/1.0.0/predict.py"
```

Comment:
Clearly, there is no predict.py in the repo. Please refer to the original repo https://github.com/neuroneural/Vox2Cortex_fork/blob/main/singularity/test.sh#L7 to understand how testing/prediction works and modify your spec.yml accordingly.

Reply:
@mselimata I also recommend testing this in the nobrainer-zoo (with the sample data we provided). The commands to test these are in the second part of the documentation provided. It goes along the lines of `nobrainer-zoo predict --model vox2cortex --data ....`. Please let us know if you have any questions about this.